dev/archery/archery/benchmark/compare.py

```diff
@@ -25,11 +25,11 @@ def items_per_seconds_fmt(value):
     if value < 1000:
         return "{} items/sec".format(value)
     if value < 1000**2:
-        return "{:.3f}k items/sec".format(value / 1000)
```

Comment:
In addition to the main change in the PR, Antoine asked me to change these units.